### PR TITLE
Move druid-api into the druid repo.

### PR DESCRIPTION
**Pull request by gianm**
This is from druid-api-0.3.17, as of commit 51884f1d05d5512cacaf62cedfbb28c6ab2535cf
in the druid-api repo.

Marking 'release notes' as the druid-api version would jump from 0.3.17 to 0.9.1. The groupId
and artifactId will remain the same.

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```
Comment (drcrallen) [inline api/pom.xml]: can you fix the header to be the standard license header instead of the old-style header?

Comment (gianm) [inline api/pom.xml]: adjusted

**Comment by drcrallen** [comment]
:+1: after travis. @gianm would you mind making a notification on the druid-api readme that the former repository is deprecated as of 0.9.1, and any patches should be submitted against this repository?

**Comment by gianm** [comment]
@drcrallen yes I can do that.

**Comment by gianm** [comment]
https://github.com/druid-io/druid-api/pull/78

**Pull request merged**